OptionSelector/OSCategoriesTableViewController.m

```diff
@@ -0,0 +1,125 @@
+//
+//  OSCategoriesTableViewController.m
+//  OptionSelector
+//
+//  Created by Daniel Distant on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "OSCategoriesTableViewController.h"
+#import "CQCategory.h"
```

Comment:
Style: You don't actually need this `#import` here, because you're not referencing the `CQCategory` class in your .h file. It should be in your .m file, though.